### PR TITLE
Add classifier and FileDataStream unit tests to test_pipeline_combining.

### PR DESCRIPTION
**Pull request by pieths**

### CODE REVIEW

File: src/python/nimbusml/tests/pipeline/test_pipeline_combining.py [hunk 1]
```diff
@@ -323,6 +323,91 @@ def test_combine_transform_and_pipeline(self):
...
+        df = transform.transform(X, as_binary_data_stream=True)
+        result_1 = predictor.predict(df)
+
+        # Combine the models and perform a prediction
+        combined_pipeline = Pipeline.combine_models(transform, predictor)
+        result_2 = combined_pipeline.predict(X)
+
+        result_2 = result_2['PredictedLabel'].astype(np.float64)
+        self.assertTrue(result_1.equals(result_2))
+        self.assertTrue(result_2.equals(y))
```
Comment (ganik) [inline src/python/nimbusml/tests/pipeline/test_pipeline_combining.py]: > self.assertTrue(result_2.equals(y)) [](start = 7, length = 36)

why would this be case?

File: src/python/nimbusml/tests/pipeline/test_pipeline_combining.py [hunk 2]
```diff
@@ -323,6 +323,91 @@ def test_combine_transform_and_pipeline(self):
...
+        result_1 = predictor.predict(df)
+
+        # Combine the models and perform a prediction
+        combined_pipeline = Pipeline.combine_models(transform, predictor)
+        result_2 = combined_pipeline.predict(infert_df)
+
+        result_2 = result_2['PredictedLabel'].astype(np.float64)
+        self.assertTrue(result_1.equals(result_2))
+        self.assertTrue(result_1.equals(infert_df['case']))
+
```
Comment (ganik) [inline src/python/nimbusml/tests/pipeline/test_pipeline_combining.py]: same here

File: src/python/nimbusml/tests/pipeline/test_pipeline_combining.py (hunk 1 above)
Comment (pieths) [reply src/python/nimbusml/tests/pipeline/test_pipeline_combining.py]: Removed in latest commit.

File: src/python/nimbusml/tests/pipeline/test_pipeline_combining.py (hunk 2 above)
Comment (pieths) [reply src/python/nimbusml/tests/pipeline/test_pipeline_combining.py]: Removed in latest commit.

**Pull request merged**